packages/dashboard/src/features/auth/components/EmailTemplateCard.tsx

```
@@ -135,6 +177,15 @@ export function EmailTemplateCard({
     }
   }, [selectedTemplate]);
 
+  const resetToDefaults = useCallback(() => {
+    if (selectedType && DEFAULT_EMAIL_TEMPLATES[selectedType]) {
+      const { subject: defaultSubject, bodyHtml: defaultBodyHtml } = DEFAULT_EMAIL_TEMPLATES[selectedType];
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)